src/Acme.UI/Infrastructure/Routing/AcmeRouter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Acme.UI.Infrastructure.Routing.VirtualRouteHandlers;
+using System.Web.Routing;
+using Umbraco.Core.Composing;
+using Umbraco.Web;
+using Umbraco.Core.Logging;
+
+namespace Acme.UI.Infrastructure.Routing
+{
+    public class AcmeRouter : IComponent
+    {
+        private ILogger _logger;
+
+        public AcmeRouter(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public void Initialize()
+        {
+            _logger.Info<AcmeRouter>("-->Initializing custom acme routing4");
+
+            /*RouteTable.Routes.MapRoute("LoginRoute1", "kamran/member/render_login/{returnUrl}",
```

Comment:
Removed commented code please

Reply:
We don't need this class. I will remove it.